### PR TITLE
add circle.yml

### PR DESCRIPTION
**Pull request by stegben**
#13

### CODE REVIEW

**Comment by stegben** [comment]
done. @ianlini plz check if any missing config

**Comment by ianlini** [comment]
you should copy the coverage report to the artifacts dir

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        Asia/Taipei
+
+    environment:
+        CIRCLE_ENV: test
+
+    post:
+        - pyenv global 2.7.10 3.5.0
+
+dependencies:
+    pre:
```
Comment (ianlini) [inline circle.yml]: 這應該用override

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        - pip install -U pip wheel && pip install -U Cython && pip install -Ur requirements.txt
+
+test:
+    override:
+        - python setup.py nosetests
+    post:
+        - mkdir $CIRCLE_ARTIFACTS/coverage
+        - cp -R coverage $CIRCLE_ARTIFACTS/coverage
```
Comment (ianlini) [inline circle.yml]: no need for the trailing `coverage` in these 2 lines

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+machine:
+    timezone:
+        Asia/Taipei
+
+    environment:
+        CIRCLE_ENV: test
+
+    post:
+        - pyenv global 2.7.10 3.5.0
```
Comment (ianlini) [inline circle.yml]: Are you sure CircleCI really use these version? 
Or it just use the system default. 
Maybe you can use `machine.python.version: 2.7.12` at this moment because we are not ready for testing multiple python versions.

Comment (stegben) [inline circle.yml]: I'll try using ```tox```

Comment (ianlini) [inline circle.yml]: That should be another PR

File: circle.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+machine:
+    timezone:
+        Asia/Taipei
+
+    environment:
+        CIRCLE_ENV: test
+
+    python:
+        version: 2.7.10
```
Comment (ianlini) [inline circle.yml]: set 2.7.12?

Comment (stegben) [inline circle.yml]: circleCI don't have 2.7.12 preinstalled

Comment (stegben) [inline circle.yml]: We can use other version in the future if we use tox
